aws/v12.3.0/release.diff

```diff
@@ -0,0 +1,67 @@
+apiVersion: release.giantswarm.io/v1alpha1			apiVersion: release.giantswarm.io/v1alpha1
+kind: Release							kind: Release
+metadata:							metadata:
+  annotations:							  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/	    giantswarm.io/docs: https://docs.giantswarm.io/reference/
+  creationTimestamp: null					  creationTimestamp: null
+  name: v12.2.0						      |	  name: v12.3.0
+spec:								spec:
+  apps:								  apps:
```

Comment:
The formatting of this file is a bit wonky. It looks like it contains a mix of spaces and tabs. I checked the previous PRs which all use spaces and are nicely displayed. It's not a big issue, but it would improve the readability of the file.

I use the following command (from devctl) to manually generate the release.diff file:
`diff aws/v12.1.4/release.yaml aws/v12.2.0/release.yaml -y -t > aws/v12.2.0/release.diff`

-t expands tabs to spaces

Reply:
I will try your command above, I think I was doing without `-t` option.